gateway/handler.go

```diff
@@ -912,13 +912,11 @@ func (i *handler) handleSuperfluousNamespace(w http.ResponseWriter, r *http.Requ
 	// - redirects to intendedURL after a short delay
 
 	w.WriteHeader(http.StatusBadRequest)
-	if err := redirectTemplate.Execute(w, redirectTemplateData{
+	_ = redirectTemplate.Execute(w, redirectTemplateData{
 		RedirectURL:   intendedURL,
 		SuggestedPath: intendedPath.String(),
 		ErrorMsg:      fmt.Sprintf("invalid path: %q should be %q", r.URL.Path, intendedPath.String()),
-	}); err != nil {
-		i.webError(w, r, fmt.Errorf("failed to redirect when fixing superfluous namespace: %w", err), http.StatusBadRequest)
```

Comment:
@hacdias instead of silently failing in all these places and ignoring `err`, could we still write the error to the `http.ResponseWriter`? 

Even tho we can't set error status code, this will be useful for end user who tries to figure out what went wrong, unlike the empty body.

My thinking is to replace `i.webError(` calls with:

```go
_, _ = w.Write([]byte(fmt.Sprintf("error during body generation: %v", err)))
```

Thoughts?

Reply:
@lidel I did, I think it is fine. My initial idea was that we know that the templates are correct. They are tested, we send the right structs. Thus, having an error _executing_ the template is extremely unlikely. Most of these errors will likely be network errors, and attempting to write again will also fail.

But yes, by printing the error we will also catch the extremely unlikely template executing errors.